src/main/resources/assets/js/application/recruiter/services/tdprPersonsService.js

```diff
@@ -1,20 +1,11 @@
 define(['application/recruiter/tdprRecruiterModule'], function (tdprRecruiterModule) {
     tdprRecruiterModule.service('tdprPersonsService', function ($http, dateFilter, $q) {
         var persons;
```

Comment:
The service should be stateless - if we want to preserve persons data - use resolve